cv_bridge/include/cv_bridge/cv_bridge.h

```diff
@@ -256,10 +428,42 @@ template<> struct Definition<cv_bridge::CvImage>
 
 template<> struct HasHeader<cv_bridge::CvImage> : TrueType {};
 
+/////////////////////////////////////// CvCompressedImage ///////////////////////////////////////////
+
+template<> struct MD5Sum<cv_bridge::CvCompressedImage>
```

Comment:
yes, remove: the proper way to publish a compresed image should be through image transport anyway